lib/compat/wordpress-7.0/class-gutenberg-rest-registered-templates-controller.php

```diff
@@ -1,7 +1,8 @@
 <?php
 
-class Gutenberg_REST_Static_Templates_Controller extends WP_REST_Templates_Controller {
```

Comment:
This is optional, but I found it confusing that the classes were named differently for the same routes. See: https://github.com/ramonjd/wordpress-develop/blob/trunk/src/wp-includes/rest-api/endpoints/class-wp-rest-registered-templates-controller.php#L3